src/interpreter/plugin/CodePlugin.ts

```diff
@@ -26,4 +32,10 @@ export class CodePlugin extends FunctionPlugin {
       return value.charCodeAt(0)
     })
   }
+
+  public unicode(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('UNICODE'), (value: string) => {
+      return value.codePointAt(0) ?? new CellError(ErrorType.VALUE, ErrorMessage.EmptyString)

```

Comment:
It's really nice. Could I ask you to refactor the `code()` to follow the same logic?